triplet/main.py

```diff
@@ -302,7 +304,8 @@ def train(train_loader, model, optimizer, epoch):
         # compute output
         out_a, out_p, out_n = model(data_a), model(data_p), model(data_n)
         loss = F.triplet_margin_loss(out_a, out_p, out_n, margin=args.margin)
-
+        if args.anchorswap:
+            loss += F.triplet_margin_loss(out_p, out_a, out_n, margin=args.margin)
```

Comment:
you forgot adding the flag 
https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L681

no need for if statement just pass the argument
```
loss += F.triplet_margin_loss(out_p, out_a, out_n, margin=args.margin, swap=args.anchorswap) 
```